### PR TITLE
Add contact form validation to Portfolio using Javascript

### PR DESCRIPTION
**Pull request by shahira-sadat**
In this PR request, I have worked on the contact form validation using javascript. 
Form validation includes the following checks:

- Inputs are empty or not
- Name input maximum character is greater less than 30 or not
- Email entered is valid or not
- Email input is in lowercase or not

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -172,51 +172,31 @@ <h2>Languages</h2>
...
+      <li><input type="email" class="form-control" name="email" id="email" placeholder="Email address">
+      </li>
+      <li> </li>
+      <li><textarea name="text" class="form-control" id="text" placeholder="Message"></textarea></li>
+      <li> </li>
+      <li class="flex-item6" id="submit-btn"><button type="submit">Get In Touch</button></li>
+      <li class="shape"><img src="image/Shape.png" alt="shape"></li>
+    </ul>
+  </form>
+</section>
```
Comment (CollinsTatang) [inline index.html:196]: - [ ] Please kindly replace the background color with the background image as per [Figmal Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
### **Figmal Design**
form-b
### **Your Design**
formYour

File: index.html [hunk 2]
```diff
@@ -172,51 +172,31 @@ <h2>Languages</h2>
...
+  </div>
+  <form action="https://formspree.io/f/xbjwrzvy" id="form" class="form" method="post">
+    <ul class="inner-list">
+      <li><input type="text" class="form-control" placeholder="Full name" name="name" id="name" size="40"
+          maxlength="30">
+      </li>
+      <li> </li>
+      <li><input type="email" class="form-control" name="email" id="email" placeholder="Email address">
+      </li>
+      <li> </li>
```
Comment (CollinsTatang) [inline index.html:189]: - [ ] The error message should show up close to the `submit` button.
### **Your current case**
emailE

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @shahira-sadat, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (shahira-sadat) [reply index.html:189]: - Fixed the background image issue
- The location of errors are okay now(Every error message is properly located)
Screen Shot 2022-06-17 at 6 44 33 PM

Comment (muneebulrehman) [reply index.html:189]: @shahira-sadat , as pointed out by the previous reviewer, the location of the error message should be close to the submit button. However, it shows on top of the text area. Here is the reference.
image

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @shahira-sadat , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- Your layout for the website is messed up. The about section is showing just below your header. Here is a reference snapshot. Consider fixing it.
image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (CollinsTatang) [reply index.html:189]: Hi @shahira-sadat 

- [ ] Please the error message for the email should show up close to the submit button.

**Comment by shahira-sadat** [comment]
@muneebulrehman [muneebulrehman](https://github.com/muneebulrehman)

The last error is next to submit button but if we have several errors then it make some problem. 
kindly check the screenshot

Screen Shot 2022-06-17 at 7 30 06 PM
.

**Comment by muneebulrehman** [comment]
Hey @shahira-sadat, you only need to have one error, and that is for email validation. You can skip the error for the missing message.

**Review by roshan-bajgain — APPROVED** [review]
Hi @shahira-sadat, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

- [ ] You can make the error message more stylist by styling it.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**